app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -30,9 +30,7 @@ const DeleteAlert = ({
       onSubmit={handleDelete}
       onClose={onClose}
       message="Are you sure you want to continue? This cannot be undone."
-      title={`Delete ${selectedNoteIds.length} ${
-        selectedNoteIds.length > 1 ? "notes" : "note"
-      }?`}
+      title={`Delete note?`}
```

Comment:
Here, I guess we are using neetoUI Modal. Can we switch to neetoUI alert?

Reply:
I went with what was being used in wheel by default. Will change.

Reply:
Actually, neetoUI Alert is being used here.
Reference: https://github.com/abrahamparayil/wheel/blob/a9e60dfe7198e547cbcdc5e75015c1b6b9e47558/app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx#L3